# timestamp (uint64,  in microseconds since system start) will be generated automatically

float32 roll	    # body angular rates in NED frame
float32 pitch	    # body angular rates in NED frame
float32 yaw	    # body angular rates in NED frame
float32 thrust	    # thrust normalized to 0..1
